crates/testsuite/features/cmd.feature

```diff
@@ -37,10 +37,15 @@ Feature: Rooch CLI integration tests
       Then assert: "{{$.move[-1].execution_info.status.type}} == executed"
 
       # session key
-      Then cmd: "session-key create  --app-name test --app-url https:://test.rooch.network --scope 0x3::empty::empty"
+      Then cmd: "session-key create --app-name test --app-url https://test-seed.rooch.network --scope 0x3::empty::empty"
       Then cmd: "move run --function 0x3::empty::empty  --session-key {{$.session-key[-1].authentication_key}}"
       Then assert: "{{$.move[-1].execution_info.status.type}} == executed"
 
+      # session key from session key
+      Then cmd: "session-key create --app-name test --app-url https://test-seed.rooch.network --session-key {{$.session-key[-1].authentication_key}} --scope 0x3::empty::empty"
+      Then cmd: "move run --function 0x3:cargo test -p testsuite --test integration:empty::empty  --session-key {{$.session-key[-1].authentication_key}}"
```

Comment:
can not run testsuite in testsuite case.

Reply:
Will remove it.